using_images/other_images/jenkins.adoc

```diff
@@ -64,6 +79,71 @@ $ oc new-app -e \
+either from data seeded into the pod, or by interacting with the {product-title} API server via REST invocations.  There are configuration fields available for the plugin, but they should only be required for perhaps
+development testing or atypical configurations when you want to access a different {product-title} installation for OAuth than the installation hosting Jenkins.
+
+The login process is comprised of a series of HTTP redirects.  Rather than the traditional Jenkins login page, you will first be directed to an {product-title} login page, similar to what you see when you log into the web console.  If login
+is successful, you will be redirected to the page within the Jenkins UI you are trying to access.
+
+Valid user / password combinations are controlled by the identity provider in use with {product-title}.  So, for example,  if the default identity provider `Allow All` is used, you can provide any
+non-empty string as the password for `admin` or any other user name of your choosing.  If one of the other identity providers is running, valid user/password combinations stored with your identity provider must be used to login.
+
+Valid users are automatically added to the Jenkins authorization matrix as part of login.
+
+The {product-title} `*Role*` of the user then dictates which permissions within the Jenkins authorization matrix are associated with the user.  The `admin` `*Role*` maps to the set of
+permissions traditionally associated with the `admin` user in the {product-title} Jenkins image:
+
+* hudson.model.Computer.Configure
+* hudson.model.Computer.Delete
+* hudson.model.Hudson.Administer
+* hudson.model.Hudson.Read
+* hudson.model.Item.Build
+* hudson.model.Item.Configure
+* hudson.model.Item.Create
+* hudson.model.Item.Delete
+* hudson.model.Item.Read
+* hudson.model.Item.Workspace
+* hudson.model.Run.Delete
+* hudson.model.Run.Update
+* hudson.model.View.Configure
+* hudson.model.View.Create
+* hudson.model.View.Delete
+* hudson.scm.SCM.Tag
+
+Users with the `edit` or `view``*Roles*` for the {product-title} project have progressively reduced permissions within Jenkins.
+
+For the `view` `*Role*`:
+
+* hudson.model.Hudson.Read
+* hudson.model.Item.Read
+
+For the `edit` `*Role*`, in addition to the permissions available to `*view*`:
+
+* hudson.model.Item.Build
+* hudson.model.Item.Configure
+* hudson.model.Item.Create
+* hudson.model.Item.Delete
+* hudson.model.Item.Workspace
+* hudson.scm.SCM.Tag
+
+Permissions for users can be changed after they are initially established.  Note, changes to permissions in Jenkins do not result in changed permissions for the users in {product-title}.
+
+Also note, the OpenShift Login plugin polls {product-title} for permissions and will update the permissions for each user currently defined in Jenkins.  You can control how often the polling
```

Comment:
need clarification around what happens when you change permissions in jenkins and then polling happens (and either the openshift permissions have changed, or have not changed but conflict with the changes made in jenkins)